client/components/mobile/Explorer.jsx

```diff
@@ -1,15 +1,19 @@
 import React from 'react';
+import { useTranslation } from 'react-i18next';
 import styled from 'styled-components';
 import PropTypes from 'prop-types';
 import Sidebar from './Sidebar';
 import ConnectedFileNode from '../../modules/IDE/components/FileNode';
 
 
-const Explorer = ({ id, canEdit, onPressClose }) => (
-  <Sidebar title="Files" onPressClose={onPressClose}>
-    <ConnectedFileNode id={id} canEdit={canEdit} onClickFile={() => onPressClose()} />
-  </Sidebar>
-);
+const Explorer = ({ id, canEdit, onPressClose }) => {
+  const { t, i18n } = useTranslation();
+  return (
+    <Sidebar title={t('MobileSideBar.Files')} onPressClose={onPressClose}>
```

Comment:
It'd be great if the first part of the translation key can be named after the component/filename. So this would be `Explorer`. This makes it easier to figure out the link between the translation language files and the components.

```suggestion
    <Sidebar title={t('Explorer.Files')} onPressClose={onPressClose}>
```

Reply:
Done